cohort-engine-api-web/src/main/java/com/ibm/cohort/engine/api/service/CohortEngineRestHandler.java

```diff
@@ -236,6 +238,7 @@
 			"<p>The <code>patientIds</code> is a comma separated list of patients to run. Supplying a single patient does not need any trailing commas.</p>" +
 			"<p>The parameter types and formats are described in detail in the <a href=\"http://alvearie.io/quality-measure-and-cohort-service/#/user-guide/parameter-formats?id=parameter-formats\">user guide</a>.</p>" +
 			"<p>The <code>loggingLevel</code> will determine how much and what type of logging to provide. The options are NA, COVERAGE, and TRACE.</p>" +
+			"<p>For more detailed information about the cohort endpoint, check out our <a href=\"http://alvearie.io/quality-measure-and-cohort-service/#/user-guide/rest_endpoints?id=cohort-endpoint\">documentation</a> here.</p>" +
```

Comment:
@shelleyMurphy : As i had pointed it out to you the other day can you add a statement or two regarding the fact that in the Example contents for the request_data.json you are showing for the cohort evaluation endpoint the measurement period interval start type and end type are different. One is date and other is datetime.  Need to clarify for users that either one can be used but the start and end should match each other for any given instance of use. You cannot have one as date type and the other as datetime type.

Reply:
I'll just make them the same value and hope that people look at the parameter type/format url